lib/utils/utils.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class Strings {
+  static const username = 'Username';
+  static const password = 'Password';
+  static const no_accont = "Don't have an account? ";
+  static const sign_up = "Sign up";
+  static const forgot_psswd = 'Forgot your password?';
+  static const login = 'LOGIN';
+  static const login_fb = 'Sign up with Facebook';
+}
+
+class App_colors {
+  static const login = 0xffff2d55;
+}
+
+class Text_styles {
+  static const textstyle = TextStyle(
+    fontSize: 15,
+    color: Colors.white,
+    fontFamily: 'SFUIDisplay',
+  );
+}
```

Comment:
use more meaningful name than "textstyle". Noone would now anything about this textstyle with such a name.

Reply:
Please then suggest me name for this

Reply:
it could be loginScreenDefault for now :)

Reply:
Ok done

Reply:
it might be AppTextStyles or TextStylesUtil :)

Reply:
ok